frontend/src/metabase/visualizations/echarts/cartesian/option/format.ts

```diff
@@ -1,83 +0,0 @@
-import type {
```

Comment:
Moved to https://github.com/metabase/metabase/pull/36879/files#diff-a925af260455ef56ff9bc97d2376ab8cc1f92bc51256d5ab0fdb4b39a9088363R323